lxd/main.go

```diff
@@ -633,6 +635,10 @@ func setupLXD() error {
 	}
 
 	if *argAuto {
+		if *argStorageBackend == "" {
```

Comment:
lxd init basically calls "lxc config unset core.https_address" and "lxc config unset core.trust_password" then sets them again based on your answers.